### ISSUE TITLE
The second image is missing.

### ISSUE THREAD

**Issue by jannahpitogo** [jannahpitogo/github_issues_prs_practice#1]
It isn't 100% accessibility in the Lighthouse report. 
Some images doesn't have an alt text.
An image is missing
